Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,16 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+ 
+
```

Comment:
good work, please read what you have been asked again, 
write a program that will take the hogwarts **array as input** and display the names of the people who belong to the Gryffindor house.

Reply:
thank you, refactored